net-analyzer/s3scanner/files/s3scanner

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd /usr/share/s3scanner
+python ./s3scanner.py $@
```

Comment:
you can add `inherit eutils` and use `make_wrapper ...` instead of this script (see my next comments below)